src/test/java/ua/com/juja/microservices/keepers/slackbot/controller/KeepersSlackCommandControllerTest.java

```diff
@@ -113,4 +114,67 @@ public void onReceiveSlashKeeperAddShouldReturnErrorMessageIfOccurException() th
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.text").value("something went wrong"));
     }
+
+    @Test
+    public void onReceiveSlashCommandGetKeeperDirectionsReturnEmptyRichMessage() throws Exception {
```

Comment:
No, I don't.
@OlgaKulikova Do we have a wiki page with mentioned conventions?
It's widely used name pattern in Gamification slack bot tests and I used it respectively.
If possible please propose your version.